llvm/lib/Target/RISCV/GISel/RISCVInstructionSelector.cpp

```diff
@@ -493,6 +498,50 @@ bool RISCVInstructionSelector::select(MachineInstr &MI) {
     MI.eraseFromParent();
     return constrainSelectedInstRegOperands(*Bcc, TII, TRI, RBI);
   }
+  case TargetOpcode::G_BRJT: {
+    // FIXME: Move to legalization?
+    unsigned EntrySize =
+        MF.getJumpTableInfo()->getEntrySize(MF.getDataLayout());
+    assert((EntrySize == 4 || (Subtarget->is64Bit() && EntrySize == 8)) &&
```

Comment:
Why do we limit EntrySize? How come we do not handle PointerSize entry or Inline entry?

Reply:
PointerSize would be 4 for RV32 and 8 for RV64 so those are handled. RISC-V doesn't use EK_Inline as far as I know.